src/platform-apps/channels/styles.scss

```diff
@@ -221,6 +221,7 @@ $scrollbar-width: 5px;
     overflow: inherit;
     transition: opacity 0.9s ease-out;
     opacity: 1;
+    clear: both;
```

Comment:
what does this do (from a css perspective) 👁️ ?

Reply:
The issue is related to using "float: left" and "float: right" for the messages. When you "float" something it takes it out of the normal layout flow so the thing below it will actually render over top of the thing you floated. The `clear` means roughly, "take into account floated things when positioning me".  That's a really bad simplification but probably good enough for now. lol

Reply:
haha ok, i got the context. thanks!